js/packages/web/src/actions/getStorefront.ts

```diff
@@ -108,7 +109,7 @@ export const getStorefront = async (
   const client = createClient({
     url: REDIS_URL,
     socket: {
-      tls: true,
+      tls: REDIS_TLS_ENABLED,
```

Comment:
I think we'll need to spread an empty object in the event of local dev and an object with `tls: true` for prod (false isn't a valid value for tls here I guess)

Reply:
Actually `tls: true` is just required for the build step to complete it seems :-/